changelogs/fragments/8355-keycloak-idp-sanitize.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
```

Hmm, I think this classifies as a security fix:
```suggestion
security_fixes:
```